docs/src/preprocess/InstructionSet/InstructionSet.js

```diff
@@ -1537,6 +1537,39 @@ halt
     "Tag checks": "",
     "Tag updates": "",
   },
+  {
+    id: "to_radix_le",
+    Name: "`TORADIXLE`",
+    Category: "Conversions",
+    Flags: [{ name: "indirect", description: INDIRECT_FLAG_DESCRIPTION }],
+    Args: [
+      {
+        name: "srcOffset",
+        description: "memory offset of word to convert.",
+      },
+      {
+        name: "dstOffset",
```

Comment:
Does it support this to be indirect? brillig provides a pointer to the result

Reply:
Yep that is the plan. The details of how the AVM will implement storing slices is still a bit up in the air